src/main/kotlin/com/jobtracker/api/repository/entities/ApplicationEntity.kt

```diff
@@ -0,0 +1,32 @@
+package com.jobtracker.api.repository.entities
+
+import java.util.UUID
+import javax.persistence.*
+
+@Entity
+@Table(name = "applications")
+data class ApplicationEntity(
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id", nullable = false)
+    val id: UUID,
+    @Column(name = "position_title", nullable = false) val positionTitle: String,
+    @Temporal(TemporalType.DATE)
+    @Column(name = "submit_date", nullable = false)
+    val submitDate: Date,
+    @Enumerated
+    @Column(name = "status", nullable = false) val status: Status
+    @Column(name = "skills", nullable = true) var skills: String?,
```

Comment:
The lengths are set to 255 unless otherwise specified, will need to set it to whatever we decided on.